RELEASE-NOTES.rst

```diff
@@ -18,6 +18,243 @@
+   order to come up with the need of a complex cataloging editor.
+-  Displays custom logos and header color for each organisation and the
+   professional interface.
+-  Adds an “history” tab in the patron profile to display transactions
+   history of the last 6 months.
+-  Adds a button in the requests tab of the patron profile to allow
+   patrons to cancel their own requests.
+-  Updates the frontpage for the pilot libraries instance, with relevant
+   information.
+
+Search and indexing
+-------------------
+
+-  Improves ebooks bulk indexing (``invenio utils reindex``,
+   ``invenio utils runindex``).
+-  Improves persons indexing during document indexing and document
+   creation.
+-  Fixes missing mappings in JSON schemas.
+-  Adds default sorting for each resource.
+
+Circulation
+-----------
+
+-  Adds a method to get the timezone of a library to improves
+   computation of a checkout due date.
+-  Links loans and fees.
+-  Enriches fees index with ``loan.pid``, ``patron.pid`` and
+   ``organisation.currency``.
+-  Adds an order parameter to sort pending loans.
+-  Improves the method for finding the correct location of an item as it
+   is in circulation.
+
+Metadata
+--------
+
+-  Improves ``dojson`` transformations (MARC21 to JSON), especially the
+   ability to do parallel transformations, which is necessary for
+   complex tasks.
+-  Updates and improves unimarc ``dojson`` transformation (BnF
+   importation).
+-  Adds comprehensive languages list to the document JSON schema.
+-  Reserves range of PIDs, usefull to import linked resources from an
+   external system. Adds a dump function to compute an unstructured
+   ``_text`` field based on structured data. This field is for display
+   purpose. This new field is also added to the JSON schema.
+-  Adds a dump function to items, to populate items index with
+   organisation, location and availability data.
+-  Fixes documents dumps for records without series.
+-  Fixes ``dojson`` series field transformation for ebooks.
+-  Adds a CLI command to automatically translate the LoC languages list.
+-  Implements and rewrites ``provisionActivity`` field into the new data
+   model and adapts the needed transformations.
+-  Fixes ``provisionActivity`` ``startDate``.
+-  Implements edition statement field into the new data model and the
+   needed transformations.
+-  Adds a command to the bootstrap script to compile JSON (JSON
+   references for definitions).
+-  Improves performance with MEF persons importations:
+
+   -  Imports MEF persons in the DB, not only in ES.
+   -  Reduces the number of requests to the DB and ES.
+
+-  Updates JSON schemas to the new ``ngx-formly`` library (form options
+   are now directly in the schema).
+-  Sets the document abstract field to textarea type.
+
+Acquisition
+-----------
+
+-  Adds new resources for vendors file, orders and order lines.
+-  Adds new resources for acquisition budget and acquisition account.
```

Comment:
Adds new resources for the organisation acquisition budget and the library acquisition accounts